components/navbar.tsx

```diff
@@ -2,10 +2,16 @@ import Link from 'next/link';
 import type { FC } from 'react';
 import { BsGithub } from 'react-icons/bs';
 import { SiDiscord, SiStorybook } from 'react-icons/si';
-import { Badge, DarkThemeToggle, Tooltip } from '~/src';
+import { Badge, Button, DarkThemeToggle, Tooltip } from '~/src';
 import pkg from './../package.json' assert { type: 'json' };
 import { DocSearchInput } from './docsearch-input';
 
+const buttonIconTheme = {
+  size: {
+    md: 'p-2.5',
```

Comment:
This points out a missing feature of `IconButton`.
That's how I arrived to this issue of ghost button.

I believe what you want is something like
```tsx
<Button ghost icon>
  <SiStorybook aria-hidden className="h-5 w-5" />
</Button>
```
Here is the official Button with icon https://flowbite.com/docs/components/buttons/#buttons-with-icon
which uses the padding `2.5`
<button type="button" class="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-full text-sm p-2.5 text-center inline-flex items-center me-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">

which